core/fast_string.rs

```diff
@@ -27,7 +27,9 @@ pub enum FastString {
   Static(&'static str),
 
   /// Created from static data, known to contain only ASCII chars.
-  StaticAscii(&'static str),
+  /// For extra optimization when snapshotting, external references should
+  /// be provided for these strings.
+  StaticAscii(&'static str, v8::OneByteConst),
```

Comment:
It was a mistake for me to make this enum accessible -- we should make it an inner private member of the FastString struct (named FastStringInner) and then we can initialize it in a method (ie: FastString::static_ascii(...))

Reply:
I'm not sure I understand what you mean. What we would initialize in a method?